app/Role.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+enum Role: int
+{
+    case USER = 0;
+    case ADMIN = 1;
+    case SUPER_ADMIN = 2;
+}
```

Comment:
Lest think about that package -> https://spatie.be/docs/laravel-permission/v6/introduction